src/apps/chifra/pkg/config/scrapeGroup.go

```diff
@@ -17,6 +17,7 @@ type ScrapeSettings struct {
 	UnripeDist   uint64 `toml:"unripeDist" json:"unripeDist"`
 	AllowMissing bool   `toml:"allowMissing" json:"allowMissing,omitempty"`
 	ChannelCount uint64 `toml:"channelCount" json:"channelCount,omitempty"`
+	NotifyUrl    string `toml:"notifyUrl" json:"notifyUrl,omitempty"`
```

Comment:
I wonder if this should be here or in its own top-level group in the config file. Question: what happens if someone has 10 chains and wants to customize the URL with other parameters for their own use case?